.gitignore

```diff
@@ -1,14 +1,17 @@
-# Ignore byte-compiled Python code.
+# Byte-compiled Python code.
```

Comment:
This file seems unrelated to this change? If it's needed due to another change, could you add it htere or pull it out into a separate PR for @duenez please?

Reply:
Yes moved out (and kept my additions consistent with the above)